src/graphql/root/subscription/my-updates.ts

```diff
@@ -102,13 +120,15 @@ type MeResolveIntraLedger = {
 
 type MeResolveSource = {
   errors: IError[]
+  realtimePrice?: MePayloadPrice
   price?: MePayloadPrice
```

Comment:
Should we add a deprecated for this or are we keeping both?

Reply:
It is part of a union really couldn't find a way to do it.